src/components/PhotosHeader.vue

```diff
@@ -0,0 +1,201 @@
+<!--
+ - @copyright Copyright (c) 2019 Louis Chemineau <louis@chmn.me>
+ -
+ - @author Louis Chemineau <louis@chmn.me>
+ -
+ - @license AGPL-3.0-or-later
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+<template>
+	<div class="photos-header">
+		<Actions v-if="selection.length === 0" :menu-title="t('photos', 'Add')" :primary="true">
+			<Plus slot="icon" />
+			<!-- <ActionButton :close-after-click="true" :title="t('photos', 'Upload media')" @click="openUploader">
```

Comment:
**suggestion (non-blocking):** Note in an inline comment which parts are placeholders explicitly for other reviewers